content/blog/2018-04-13-rprofile-noam-ross.md

```diff
@@ -0,0 +1,80 @@
+---
+slug: rprofile-noam-ross
+title: ".rprofile: Noam Ross"
+date: 2018-04-13
+preface: "In this occasional series, we interview someone using a loosely defined set of interview questions for the purpose of demystifying the creative and development processes of R community members. This interview was conducted by Sean Kross and prepared by Kelly O'Briant as part of an rOpenSci unconf17 project."
+authors:
+  - name: Kelly O'Briant
+    url: https://kellobri.github.io/
+categories: blog
+topicid:
+tags:
+- R
+- community
+- interviews
+- rprofile
+---
+
+<img src="/img/blog-images/2018-04-13-rprofile-noam-ross/noam-ross.jpg" alt="Noam Ross" style="margin: 0px 20px; width: 250px;" align="left">
+_Dr. Noam Ross [@noamross on Twitter] is a disease ecologist at [EcoHealth Alliance](https://www.ecohealthalliance.org/) in NYC, as well as an editor for rOpenSci. Topics of discussion included Noam's history with R and rOpenSci, working in a team-driven research environment, and inspirations for pushing research processes and rOpenSci projects in exciting new directions._
+
+---
+
+KO: What is your name, job title, and how long have you been using R?
```

Comment:
should these be SK & KO, given that "This interview was conducted by Sean Kross and prepared by Kelly O'Briant..." and same for the other one we already published?

Unless you object, I will tag Sean in the tweet about this one

Reply:
I used Dr. Julia's interview as a template and neglected to change the preface text back. Sean wasn't involved in this interview - thanks for catching!